g2p/cli.py

```diff
@@ -56,51 +57,95 @@ def cli():
 @click.option(
     "--ipa/--no-ipa", default=False, help="Generate mapping from LANG-ipa to eng-ipa."
 )
+@click.argument(
+    "out_lang",
+    required=False,
+    default=None,
+    #type=click.Choice([x for x in LANGS_NETWORK.nodes if is_ipa(x)]),
+    type=str,
+)
 @click.argument(
     "in_lang",
-    type=click.Choice(
-        [x for x in LANGS_NETWORK.nodes if not is_ipa(x) and not is_xsampa(x)]
-    ),
+    #type=click.Choice([x for x in LANGS_NETWORK.nodes if not is_ipa(x) and not is_xsampa(x)]),
```

Comment:
If we're replacing this - I think we can remove this instead of uncommenting it.

Reply:
you're right. done